hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/util/FilterGenVisitor.java

```diff
@@ -42,9 +43,10 @@ private String quoteStringLiteral(String value) {
     }
   }
 
-  private String visitAnd(Expression left, Expression right) {
-    String leftResult = left.accept(this);
-    String rightResult = right.accept(this);
+  @Override
+  public String visitAnd(Predicates.And and) {
```

Comment:
Is case sensitivity same between hive-sync and spark integration ?

Reply:
hive sync is always case insensitive

Reply:
Sounds good.